subsys/ipc/ipc_service/lib/Kconfig

```diff
@@ -11,12 +11,13 @@ config IPC_SERVICE_STATIC_VRINGS
 	help
 	  "Static VRINGs library"
 
-config IPC_SERVICE_STATIC_VRINGS_ALIGNMENT
+config IPC_SERVICE_STATIC_VRINGS_MEM_ALIGNMENT
 	int "VRINGs alignment"
 	depends on IPC_SERVICE_STATIC_VRINGS
-	default 4
+	default 32
```

Comment:
What about setting default to 4 for compatibility with legacy?

Reply:
What if one peer uses cache and the other one does not? The defaults would differ and the communication could not be established out of the box

Reply:
aaaalright, switching to `4`.